pkg/admission/karydia/admission_pod.go

```diff
@@ -128,18 +128,36 @@ func mutatePodSecurityContext(pod corev1.Pod, setting Setting, patches Patches)
 				},
 			})
 			annotatePod(pod, &patches, "karydia.gardener.cloud/podSecurityContext.internal", setting.src+"/"+setting.value)
+		} else if secCtx.RunAsUser == nil && secCtx.RunAsGroup == nil {
+			patches.operations = append(patches.operations, patchOperation{
+				Op:    "add",
+				Path:  "/spec/securityContext/runAsUser",
+				Value: uid,
```

Comment:
For consistency, pass by pointer: Reference variables with `&uid` as in the coding before.